src/plugins/userVoiceShow/index.tsx

```diff
@@ -73,28 +88,69 @@ export default definePlugin({
     authors: [Devs.LordElias],
     settings,
 
+    start() {
+        VoiceStateStore.getAllVoiceStates(); // reduces api spam i hope
```

Comment:
```suggestion
        VoiceStateStore.getAllVoiceStates(); // reduces api spam i hope
```

This returns an object and makes no API calls

Reply:
So i can remove that line and be fine?

Reply:
yes